netdata-installer.sh

```diff
@@ -413,6 +427,66 @@ fi
 
 trap build_error EXIT
 
+# -----------------------------------------------------------------------------
+
+fetch_libmosquitto() {
+  download_tarball "${1}" "${2}" "libmosquitto" "cloud"
```

Comment:
I've removed the cpp build step in the mosquitto repo. Please update this to pull a version with the new commit so that we don't get the cpp build errors in the Netdata installer.